### DISCUSSION TITLE
Excluding paths/tags/notes from card scan

### DISCUSSION THREAD

**Discussion by Bigmedrepo** [ObsidianToAnki/Obsidian_to_Anki#279] [Ideas]
It would be amazing to have the ability to exclude a certain folder from the scanning/rescanning process for cards.
I enjoy making cards in obsidian because it gives me the ability to create links to it from the cards but at the same time it's a huge drag going back to the Obsidian to correct a minor spelling error or to add a small note to the card and I prefer just doing these things as I study the cards in Anki.
I know the plugin doesn't rescan files that haven't had any new changes and so I make separate notes for making cards but then if I change the title to one of the notes that have been linked to in a card, the Anki card note will be altered and the plugin will scan it, thus nullifying any changes I would have made in Anki.
If such a feature existed I could simply exclude any note or folder from the scan, saving me a lot of trouble.
There are also other uses people can make from this, I'm sure.
Anyways, thanks.

**Comment by brendanterry** [comment]
Relatedly, the ability to designate a subfolder in my vault for the scan, rather than scanning the entire vault, would be ideal for my use case.
